docs/user-guide/basic-usage.md

```diff
@@ -2,34 +2,30 @@
 
 ## System Requirements
 
-Building Styx requires Java 1.8. It can be built with 1.8.0_45. Earlier maintenance
-releases may work, but are not guaranteed to.
-
-The build system requires Apache Maven. The Styx team uses Maven version 3.2.1
-for automated continuous integration builds. On Mac OSX, a version installed 
-by HomeBrew is satisfactory.
+Running Styx requires Java 1.8. It can be run with 1.8.0_45 or later versions. Earlier maintenance
```

Comment:
Why is the remark about maven being removed?

Reply:
The rest of the instructions were considering that we were downloading an already compiled binary, but here we were explaining how to build the app instead of how to run it.

Probably we can also add instructions to  build from source code.

Reply:
Ok. Makes sense.